pkg/internalapis/cnsvolumeoperationrequest/v1alpha1/register.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Could you use the same `register.go` file in internalapis instead of creating a new one? We have all the APIs registered in the same file.

Reply:
Makes sense, resolving this comment